FontoXPathCSharp/Expressions/ExecutionSpecificStaticContext.cs

```diff
@@ -137,11 +138,15 @@ public List<ResolvedFunction> GetResolvedFunctions()
         var knownNamespaceUri = StaticallyKnownNamespaceUtils.GetStaticallyKnownNamespaceByPrefix(prefix);
         if (knownNamespaceUri != null) return knownNamespaceUri;
 
+        _executionContextWasRequired = true;
+
         var uri = _namespaceResolver(prefix);
 
-        if (prefix != null && !_referredNamespaceByName.ContainsKey(prefix) && uri != null)
+        if (!_referredNamespaceByName.ContainsKey(prefix) && uri != null)
```

Comment:
Github said this: `Possible null reference argument for parameter 'key' in 'bool Dictionary<string, (string, string)>.ContainsKey(string key)'.` is that ok?